frontend/src/pages/MainPage.tsx

```diff
@@ -0,0 +1,11 @@
+import Header from "../components/Header";
+import Gallery from "../components/Gallery";
+
+export default function MainPage(){
+    return(
+        <div className="mainPage">
+            <Header name="Animal Zoo API"/>
```

Comment:
Here you use your Header. If you want to use it for more Page Layouts keep it. Otherwise you can add the CSS here for an h1.